docker/api/Dockerfile

```diff
@@ -34,17 +35,18 @@ COPY client.html /opt/cnaas/static
 RUN unlink /etc/nginx/sites-enabled/default
 RUN ln -s /etc/nginx/sites-available/nginx_app.conf /etc/nginx/sites-enabled/default
 RUN chown www-data:www-data /var/log/nginx
-RUN chown www-data:www-data /etc/cnaas-nms/repository.yml
+RUN chown www-data:www-data /etc/cnaas-nms/*.yml
 RUN chown -R  www-data:www-data /var/log/nginx/
 RUN chown -R www-data:www-data /var/lib/nginx
 RUN chown www-data:www-data /var/lib/nginx/
+RUN chown -R www-data:www-data /etc/nginx/
 # Give permission for API to clone/sync repos
-RUN chown www-data:www-data /opt/cnaas
+RUN chown www-data:www-data -R /opt/cnaas
```

Comment:
chown root:www-data
chmod -R u=rwX,g=rX,o=
and remove the two following lines?